src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/crud-table/crud-table.component.ts

```diff
@@ -131,7 +142,27 @@ export class CRUDTableComponent implements OnInit {
   updateSelection(selection: CdTableSelection) {
     this.selection = selection;
   }
+
   setExpandedRow(event: any) {
     this.expandedRow = event;
   }
+
+  authExport() {
+    let entities: string[] = [];
+    this.selection.selected.forEach((row) => entities.push(row.entity));
+    this.cephUserService.export(entities).subscribe((data: string) => {
+      const modalVariables = {
+        titleText: $localize`Ceph user export data`,
+        buttonText: $localize`Finish`,
```

Comment:
and probably this should be a secondary button rather than a primary.
```suggestion
        buttonText: $localize`Close`,
```